staging/src/k8s.io/client-go/rest/config.go

```diff
@@ -71,10 +71,6 @@ type Config struct {
 	// TODO: demonstrate an OAuth2 compatible client.
 	BearerToken string
 
-	// CacheDir is the directory where we'll store HTTP cached responses.
-	// If set to empty string, no caching mechanism will be used.
-	CacheDir string
```

Comment:
Note this removes this field, but the only use I could find of it was in kubectl.